extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -189,6 +189,14 @@
 						<li><?= wfMessage( 'user-identity-box-zero-state-gender' )->escaped(); ?></li>
 					<? endif; ?>
 				<? endif; ?>
+				<? if ( !empty( $user['bio'] ) ): ?>
+					<li id="discussion-bio"><?= wfMessage( 'user-identity-bio', $user['bio'] )->plain(); ?></li>
```

Comment:
Well, that's embarrassing. Thanks for catching that @Grunny. Fixed here: https://github.com/Wikia/app/pull/12337/commits/31a725b4e1a044909649b958f415f9d0c783d6e3.

http://jsutterfield.james.wikia-dev.us/wiki/User:James_test_en